### PR TITLE
Fix bug where lo units were not being properly converted

### LINKED ISSUE #3595: qubit_lo_range and meas_lo_range are still in GHz in the configuration
### Information

- **Qiskit Terra version**:
- **Python version**:
- **Operating system**:

### What is the current behavior?
The qubit and meas lo ranges are in Hz rather than GHz

### PR DESCRIPTION
**Pull request by taalexander**
### Summary
Closes #3595, #3596. Fixes a bug where lo units are not being properly converted, nor were they being accepted in Hz by `assemble`/`execute` and `schedule_los`.

### CODE REVIEW

File: qiskit/providers/models/backendconfiguration.py [hunk 1]
```diff
@@ -325,13 +325,20 @@ def __init__(self,
         self.n_uchannels = n_uchannels
         self.u_channel_lo = u_channel_lo
         self.meas_levels = meas_levels
-        self.qubit_lo_range = qubit_lo_range
-        self.meas_lo_range = meas_lo_range
+        self.qubit_lo_range = [[range[0] * 1e9, range[1] * 1e9] for range in qubit_lo_range]
```
Comment (lcapelluto) [inline qiskit/providers/models/backendconfiguration.py]: it's a little confusing seeing `range` as a variable and not the built in `range` method

**Comment by lcapelluto** [comment]
With this PR, `assemble` and `execute` will take frequency in terms of Hz, and output a qobj still in GHz, correct?

File: qiskit/providers/models/backendconfiguration.py (hunk 1 above)
Comment (taalexander) [reply qiskit/providers/models/backendconfiguration.py]: agreed

Comment (ajavadia) [reply qiskit/providers/models/backendconfiguration.py]: @taalexander so can you fix it. also the next line.

Comment (taalexander) [reply qiskit/providers/models/backendconfiguration.py]: yes, I will return to this soon.

Comment (taalexander) [reply qiskit/providers/models/backendconfiguration.py]: Fixed in 555f7272

**Comment by taalexander** [comment]
This should be good to go now.

File: qiskit/compiler/assemble.py [hunk 2]
```diff
@@ -77,24 +77,24 @@ def assemble(experiments,
...
-            List of default meas LO frequencies. Will be overridden by
-            `schedule_los` if set.
+            List of default measurement lo frequencies in Hz. Will be overridden
+            by `schedule_los` if set.
 
         qubit_lo_range (list):
-            List of drive LO ranges used to validate that the supplied qubit LOs
-            are valid.
+            List of drive lo ranges each of form `[range_min, range_max]` in Hz.
+            Used to validate that the supplied qubit los are valid.
```
Comment (lcapelluto) [inline qiskit/compiler/assemble.py]: Has this been merged or rebased? I thought I had removed the redundancy in these lines a while ago
```suggestion
            Used to validate the supplied qubit frequencies.
```

File: qiskit/compiler/assemble.py [hunk 3]
```diff
@@ -77,24 +77,24 @@ def assemble(experiments,
...
 
         meas_lo_range (list):
-            List of meas LO ranges used to validate that the supplied measurement LOs
-            are valid.
+            List of measurement lo ranges each of form `[range_min, range_max]` in Hz.
+            Used to validate that the supplied qubit los are valid.
```
Comment (lcapelluto) [inline qiskit/compiler/assemble.py]:
```suggestion
            Used to validate the supplied qubit frequencies.
```

File: qiskit/compiler/assemble.py [hunk 4]
```diff
@@ -77,24 +77,24 @@ def assemble(experiments,
             Random seed to control sampling, for when backend is a simulator
 
         qubit_lo_freq (list):
-            List of default qubit LO frequencies. Will be overridden by
+            List of default qubit lo frequencies in Hz. Will be overridden by
```
Comment (lcapelluto) [inline qiskit/compiler/assemble.py]: Why lowercase?

File: qiskit/compiler/assemble.py [hunk 5]
```diff
@@ -259,9 +259,9 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
                     for lo_config in schedule_los]
 
     if not qubit_lo_freq and hasattr(backend_default, 'qubit_freq_est'):
-        qubit_lo_freq = [freq / 1e9 for freq in backend_default.qubit_freq_est]
+        qubit_lo_freq = backend_default.qubit_freq_est
```
Comment (lcapelluto) [inline qiskit/compiler/assemble.py]: This later gets passed to `assemble_schedules`, right?

File: qiskit/compiler/assemble.py [hunk 6]
```diff
@@ -77,24 +77,24 @@ def assemble(experiments,
...
         meas_lo_freq (list):
-            List of default meas LO frequencies. Will be overridden by
-            `schedule_los` if set.
+            List of default measurement lo frequencies in Hz. Will be overridden
+            by `schedule_los` if set.
 
         qubit_lo_range (list):
-            List of drive LO ranges used to validate that the supplied qubit LOs
-            are valid.
+            List of drive lo ranges each of form `[range_min, range_max]` in Hz.
```
Comment (lcapelluto) [inline qiskit/compiler/assemble.py]: 👍 good clarification

File: qiskit/compiler/assemble.py (hunk 5 above)
Comment (taalexander) [reply qiskit/compiler/assemble.py]: yes

**Pull request merged**